documentation/docs/05-modules.md

```diff
@@ -71,6 +71,8 @@ The stores themselves attach to the correct context at the point of subscription
 - `page` contains an object with the current [`url`](https://developer.mozilla.org/en-US/docs/Web/API/URL), [`params`](#loading-input-params), [`stuff`](#loading-output-stuff), [`status`](#loading-output-status) and [`error`](#loading-output-error).
 - `session` is a [writable store](https://svelte.dev/tutorial/writable-stores) whose initial value is whatever was returned from [`getSession`](#hooks-getsession). It can be written to, but this will _not_ cause changes to persist on the server — this is something you must implement yourself.
 
+- `updated` is a [writable store](https://svelte.dev/tutorial/writable-stores) whose initial value is false. The app's version will be checked every minute for changes, setting the value to true if the version does not match the version at the time the store was created. It can be written to when custom logic is required to detect updates. There is a `check` function to force the version check immediately, returning the result.
```

Comment:
This should be updated with the new default behavior.

`updated` should be added to the list of `import`s above from `'$app/stores'`.

For consistency with the other elements in this list, we should remove the newline before `- updated`.

Reply:
I didn't actually clock that the store was writable. I don't think that's a good idea, I think it should be a readable store with a `check` method

Reply:
```suggestion
- `updated` is a [readable store](https://svelte.dev/tutorial/readable-stores) whose initial value is false. If [`version.pollInterval`](#configuration-version) is a non-zero value, SvelteKit will poll for new versions of the app and update the store value to `true` when it detects one. `updated.check()` will force an immediate check, regardless of polling.
```

Reply:
The only reason I made it writable is to enable people to add custom checks while still using that store. Since the store is used when routing errors occur, that behavior can be manually triggered.